examples/tensorflow/README.md

```diff
@@ -0,0 +1,59 @@
+## Tensorflow Model
+### Overview
+
+This model uses Tensorflow to traing an image classifier on the MNIST handdrawn numbers dataset
+
+**Project structure:**
+```
+tensorflow/
+├── README.md
+├── objects
+│   └── image.png
+├── cnn_mnist.py
+├── main.py
+└── requirements.txt
+```
+
+### Instructions
+
+In a terminal shell run:
+```bash
+# install tensorflow & other requirements
+$ pip install requirements.txt
```

Comment:
Need to change to 
`$ pip install -r requirements.txt`

Reply:
Done